source/MRMesh/MRObjectMeshHolder.cpp

```diff
@@ -300,6 +300,11 @@ void ObjectMeshHolder::applyScale( float scaleFactor )
     setDirtyFlags( DIRTY_POSITION );
 }
 
+bool ObjectMeshHolder::valid() const
+{
+    return mesh_ && mesh_->topology.numValidVerts() != 0 && mesh_->topology.numValidFaces() != 0;
```

Comment:
valid faces is enough i think (some plugins like `Mesh to PointCloud` may need only vertices)

Reply:
Fixed.